dask/dataframe/core.py

```diff
@@ -3462,7 +3461,12 @@ def _cum_agg(
             # cumulate each partitions
             name1 = f"{self._token_prefix}{op_name}-map"
             cumpart = map_partitions(
-                chunk, self, token=name1, meta=self, **chunk_kwargs
+                chunk,
+                self,
+                token=name1,
+                meta=self,
+                **chunk_kwargs,
+                enforce_metadata=False,
```

Comment:
When using `map_partitions` internally we sometimes rely on the data not being cast to a Series. The cumulative aggregation logic becomes quite cumbersome otherwise. I hope this is not introducing any other weird artefacts 🤞